### PR TITLE
Upgrade ember and page object

### PR DESCRIPTION
**Pull request by ro0gr**
This PR is a test of ember page-object [codemod](https://github.com/ro0gr/ember-page-object-codemod). The only manual [change](https://github.com/chrislopresto/ember-freestyle/commit/0373a9d09a5196a7edb867b01bd5fa9e242961eb) required was to convert `forEach()` to a `for` loop to work with `async`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,33 +33,37 @@
...
+    "ember-cli": "~3.2.0",
     "ember-cli-autoprefixer": "^0.8.1",
     "ember-cli-dependency-checker": "^2.0.0",
     "ember-cli-eslint": "^4.2.1",
     "ember-cli-github-pages": "0.2.0",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-page-object": "^1.13.0-alpha.1",
-    "ember-cli-qunit": "^4.1.1",
+    "ember-cli-page-object": "1.15.0-beta.3",
```
Comment (ro0gr) [inline package.json]: using of beta is required to use page objects with `async`/`await`

Comment (ro0gr) [reply package.json]: Mm.. seems like it currently works only with page object master including https://github.com/san650/ember-cli-page-object/pull/416.
Will update here once page object is published

Comment (chrislopresto) [reply package.json]: 👍 Thanks!

**Comment by ro0gr** [comment]
Just rebased on top of master and upgraded to ember-cli-page-object@1.15.0.

Also had to work-around a false negative eslint complaint in the last commit.

**Comment by chrislopresto** [comment]
Closing this because I just merged the rebased version #235 

Thanks so much, @ro0gr!

**Pull request closed without merging**